addon/components/bootstrap-inputs/-search.js

```diff
@@ -0,0 +1,78 @@
+import Ember from 'ember';
+import layout from '../../templates/components/bootstrap-inputs/-search';
+import { PropTypes } from 'ember-prop-types';
+import { BuilderForPropTypes, BuilderForPropDefaults } from 'ember-bootstrap-controls/utils/prop-definition-tools';
+import autoCompleteTypes from '../../utils/auto-complete-types';
+
+export const propDefinitions = {
+  autocomplete: {
+    default: 'on',
+    description: 'Indicates whether the value can be automatically completed by the browser.',
+    type: PropTypes.oneOf(autoCompleteTypes),
```

Comment:
I don't like this because autocomplete values are specific to each input type.

Reply:
addressed